### PR TITLE
added downtime and uptime messages (CU-86du6rgt5)

### PR DESCRIPTION
**Pull request by Darrickg**
Added messages in message clients page for easy copy pasting

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-86du6rgt5 Have the downtime/uptime messages easily copy-able from PA Message Clients](https://app.clickup.com/t/86du6rgt5)

File: src/views/MessageClients.jsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ function MessageClients(props) {
     alignContent: 'flex-start',
     maxWidth: '1600px',
     height: '100vh',
+    overflow: 'auto',
```
Comment (micahdbak) [inline src/views/MessageClients.jsx]: I recommend `overflowY: 'scroll'`

File: src/views/MessageClients.jsx [hunk 2]
```diff
@@ -94,6 +95,18 @@ function MessageClients(props) {
...
+          <h4>Downtime Messages</h4>
+          <p>
+            Notice: Your Brave Buttons System is down for maintenance. During this time, Button presses will not trigger an alert. You will receive
+            another text message when everything is back online. Thank you for your patience. Have a nice day.
+          </p>
+          <p>
+            Notice: Your Brave Sensor System is down for maintenance. During this time, you may not receive bathroom alerts. You will receive another
+            text message when everything is back online. Thank you for your patience. Have a nice day!
+          </p>
+          <h4>Uptime Messasges</h4>
```
Comment (micahdbak) [inline src/views/MessageClients.jsx]: Messasges -> Messages

**Comment by micahdbak** [comment]
@Darrickg push 🥺

**Review by micahdbak — APPROVED** [review]

**Pull request merged**